ccd-definition/ComplexTypes/AllocationDecision.json

```diff
@@ -51,5 +53,33 @@
     "ElementLabel": "Is allocation proposal present?",
     "SecurityClassification": "Public",
     "FieldShowCondition": "allocationProposalPresent=\"DO_NOT_SHOW\""
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "AllocationDecision",
+    "ListElementCode": "judgeLevelRadio",
+    "FieldType": "YesOrNo",
+    "ElementLabel": "Is this the right level of judge?",
+    "SecurityClassification": "Public",
+    "FieldShowCondition": "allocationProposalPresent!=\"No\""
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "AllocationDecision",
+    "ListElementCode": "proposalCorrected",
+    "FieldType": "FixedRadioList",
+    "FieldTypeParameter": "AllocationProposalList",
+    "ElementLabel": "Which level of judge is needed for this case?",
+    "FieldShowCondition": "judgeLevelRadio=\"No\"",
+    "SecurityClassification": "Public"
```

Comment:
I am not sure if this is desired - we are effectively creating different field to store the same data, if we output the data to MI / other system it would be annoying to make the mappings. 

Have you tried something like: 
"FieldShowCondition": "allocationProposalPresent!=\"Yes\" OR judgeLevelRadio=\"No\"" on proposalReason and the proposal itself? 

Another small problem is, if we select "Yes" we don't display the data at all in the tabs, (only the question), should we copy the fields in the controller maybe? Happy to discuss! You could check with BAs what's expected here.

Reply:
yes this makes sense, trying this now :)

Reply:
i've changed the tabs in the controller from speaking to Mia and Moray